### PR TITLE
Docs: Some documentation tweaks

### PR DESCRIPTION
**Pull request by ryan-roemer**
- Update docs in light of examples refactor.
- Minor language tweaks.
- Add `spectacle-mdx-loader` discussion in extensions.
- Switch to `task/rewrite` instead of `master` for some things that only exist here for validation. We're going to have to do a massive grep on `task/rewrite` before publishing anyways, I think that's OK.

### CODE REVIEW

File: docs/content/basic-concepts.md [hunk 1]
```diff
@@ -25,16 +25,15 @@ $ npm install --save spectacle
...
-For complete examples of each presentation type, please see the [`examples/`](../../examples/README.md) dir.
+<a name="mdx-or-markdown"></a>
 
-<a name="mdx"></a>
+### MDX/Markdown
 
-### MDX
-
-This approach involves statically generating your slides from a `.mdx` or .`md` file, which is accomplished using the [`spectacle-cli`](ttps://www.github.com/FormidableLabs/spectacle-cli). With the CLI, you can either generate a new presentation (`spectacle-cli spectacle-boilerplate`) or you can serve up an existing Markdown/MDX file as a presentation (`spectacle -s`). It can be installed globally, locally, or used via `npx`.
+This approach involves statically generating your slides from a `.mdx` or .`md` file, which is accomplished with [`spectacle-cli`](ttps://www.github.com/FormidableLabs/spectacle-cli). With this package, you can either generate a new presentation (with the `spectacle-boilerplate` tool) or you can serve up an existing MDX/Markdown file as a presentation (with `spectacle -s`). It can be installed globally, locally, or used via `npx`.
```
Comment (carlos-kelly) [inline docs/content/basic-concepts.md]: we are missing an h here. `(ttps://www.git`

File: docs/content/basic-concepts.md [hunk 2]
```diff
@@ -106,18 +114,18 @@ To create a Spectacle presentation that lives in a single HTML page, you will on
...
     <${Deck} theme=${customTheme}>
-        <${Slide}>
-            <${FlexBox} height="100%" flexDirection="column">
-              <${Heading} fontSize="150px">SPECTACLE</Heading>
-              <${Heading} fontSize="h2">A ReactJS Presentation Library</Heading>
-            <//>
-          <//>
+      <${Slide}>
+        <${FlexBox} height="100%" flexDirection="column">
+          <${Heading} fontSize="150px">SPECTACLE</Heading>
```
Comment (carlos-kelly) [inline docs/content/basic-concepts.md]: Should we do the `<//>` instead of the JSX closing tag?

Comment (ryan-roemer) [reply docs/content/basic-concepts.md]: Yep! A related note, I'm looking at follow on work for htm@3 which I think has real close tags!

**Comment by ryan-roemer** [comment]
Thanks for the review @carlos-kelly ! Ready for re-review

**Comment by ryan-roemer** [comment]
Released in `spectacle@6.0.0-alpha.7` (fixes to one-page.html)

**Pull request merged**